src/test/ui/methods/issues/issue-90315.rs

```diff
@@ -1,7 +1,54 @@
 fn main() {
```

Comment:
Maybe add this `#![allow(unused)]` to not require anything inside the for loops.